update docs about TF importer and accelerators (#1609)
openvinotoolkit/model_server: docs/accelerators.md

```

> GPU device can be used also on Windows hosts with Windows Subsystem for Linux 2 (WSL2). In such scenario, there are needed extra docker parameters like below:

Reviewer: ```suggestion
GPU device can be used also on Windows hosts with Windows Subsystem for Linux 2 (WSL2). In such scenario, there are needed extra docker parameters. See the command below:
```

```bash

> docker run --rm -it  --device=/dev/dxg --volume /usr/lib/wsl:/usr/lib/wsl --group-add=$(stat -c "%g" /dev/dri/render* | head -n 1) -u $(id -u):$(id -g) \

Reviewer: Can we break down changes between Unix and WSL version of `docker run` command? Tell users that `--device` changes and that mounting `/usr/lib/wsl` is needed and for what.

## Using NVIDIA Plugin

> OpenVINO Model Server can be used also with NVIDIA GPU cards. It is implemented thanks to NVIDIA plugin from the [github repo openvino_contrib](https://github.com/openvinotoolkit/openvino_contrib/tree/master/modules/nvidia_plugin).

Reviewer: ```suggestion
OpenVINO Model Server can be used also with NVIDIA GPU cards by using NVIDIA plugin from the [github repo openvino_contrib](https://github.com/openvinotoolkit/openvino_contrib/tree/master/modules/nvidia_plugin).
```
